spark3/src/main/java/org/apache/spark/sql/connector/write/SupportsPushDownRowLevelFilters.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.spark.sql.connector.write;
+
+import org.apache.spark.sql.sources.Filter;
+
+public interface SupportsPushDownRowLevelFilters extends RowLevelOperationsBuilder {
```

Comment:
Ideally, we should get rid of this if there is a good idea how to reuse similar interfaces for scans.

Reply:
I think if we use a `ScanBuilder` created by `ReplaceBuilder.asScanBuilder` then we should be able to use the one for scans.